.travis.yml

```diff
@@ -5,8 +5,10 @@ script: bash -ex .travis-opam.sh
 env:
   global:
   - PACKAGE=opam-file-format
+  - DEPOPTS=dune
+  - DUNE_PROFILE=dev
   matrix:
-  - OCAML_VERSION=4.02
+  - OCAML_VERSION=4.02 DUNE_PROFILE=release
```

Comment:
This is necessary because the API documentation includes doc comments for polymorphic variant constructors which trigger an "unattached doc comment" warning in 4.02, which was fixed in 4.03.